docs/recipes/_posts/2021-10-30-Simple_salmon_filet.html

```diff
@@ -0,0 +1,44 @@
+---
+layout: recipe
+title: "Simple salmon filet"
+tags: quick dish fish
+author: Gogo
+preview_picture: "/assets/images/salmon_filet.jpg"
+ingredients:
+    servings: 2
+    list:
+        - name: salmon filet (You can use frozen one)
+          quantity: 2
+        - name: ryour favourite ice
+          quantity: 120
+          unit: g
+        - name: shallots (2 big one)
+          quantity: 80
+          unit: g
+        - name: white wine
+          quantity: 15
+          unit: cl
+        - name: fresh cream
+          quantity: 4
+          unit: tbsp
+excerpt_separator: <!--end excerpt-->
+
+---
+
+A very simple fish dish
+
+<h2>Instructions</h2>
+<div class="recipe-instructions">
+  <ol>
+    <li>Boil water in a pot for the rice</li>
+    <li>Chop the shallots finely</li>
+    <li>Put your salmon in a gratin dish. Season with salt, pepper and some of the shallots. Cover the dish with Cellophane</li>
```

Comment:
```suggestion
    <li>Put your salmon in a gratin dish. Season with salt, pepper and some of the shallots. Cover the dish with plastic wrap</li>
```

Reply:
I'm sorry but Ryan Gosling said it : https://www.youtube.com/watch?v=Ii1p5K_BopI